automatic/nssm/legal/VERIFICATION.txt

```diff
@@ -0,0 +1,21 @@
+VERIFICATION
+
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+
+Package can be verified like this:
+
```

Comment:
The location where the binary file is located should also be inside the VERIFICATION.txt file (ie `http://www.nssm.cc/download`), and not just the full URL to the binary file.